packages/toc/src/generators/notebook/get_code_cell_heading.ts

```diff
@@ -22,16 +22,23 @@ type onClickFactory = (line: number) => () => void;
  * @param executionCount - execution count
  * @param lastLevel - last heading level
  * @param cellRef - cell reference
+ * @param index - index of referenced cell relative to other cells in the notebook
  * @returns notebook heading
  */
 function getCodeCellHeading(
   text: string,
   onClick: onClickFactory,
   executionCount: string,
   lastLevel: number,
-  cellRef: Cell
+  cellRef: Cell,
+  index: number = -1
 ): INotebookHeading {
   let headings: INotebookHeading[] = [];
+  if (index == -1) {
```

Comment:
Hm, I didn't think this would lint. Could you please use `===` instead of `==`. We try to consistently avoid the type coercion behaviors in JS unless we explicitly want them.

Reply:
Thanks for pointing this out. It should be fixed now.